lib/sandbox.js

```diff
@@ -48,28 +40,27 @@ Sandbox.prototype.start = function() {
     console.log('sandbox server created'); // emit data so that sandcastle knows sandbox is created
   });
 
-  this.server.on('error', function() {
-    _this.server.removeAllListeners();
-    _this.server = null;
-    setTimeout(function() { _this.start(); }, 500);
+  this.server.on('error', () => {
+    this.server.removeAllListeners();
+    this.server = null;
+    setTimeout(function() { this.start(); }, 500);
```

Comment:
if you are going to switch to `this`, doesn't this need to become a `=>` function?

Reply:
this got missed in this version, fixed for multiple